05-onchain.md

```diff
@@ -513,7 +513,10 @@ A local node:
   using the revocation private key.
   - SHOULD extract the payment preimage from the transaction input witness, if
   it's not already known.
-  - MAY use a single transaction to *resolve* all the outputs.
+  - if `option_anchor_outputs` applies:
+    - MUST *resolve* the _local node's main output_ in its own penalty transaction
```

Comment:
Perhaps this should be "should" instead? As there're other ways to defend against this, for example: sweep every output in its own transaction.

Reply:
Reworded, it's used now a `MAY use a single transaction to *resolve* all the outputs` which by the negative allow to sweep every output in its own transaction as _first_ broadcast to mitigate. I kept the MUST if the confirmation isn't reached near to CSV expiration as suggested by Johan. Thus you're free to aggregate as you like until some point to save on fees but after it's a matter of security and should you care to confirm.